client/package.json

```diff
@@ -67,7 +67,7 @@
     "@babel/preset-env": "^7.7.6",
     "@babel/preset-react": "^7.7.4",
     "@rollup/plugin-eslint": "^8.0.1",
-    "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git#v0.5.1",
+    "@tacc/core-styles": "https://gitpkg.now.sh/TACC/tup-ui/libs/core-styles?tup-starter-nx--core-styles-integration",
```

Comment:
Definitely see https://github.com/TACC/Core-CMS/pull/495#discussion_r887157267.

Reply:
Now (1624401) Core Styles is installed via NPM at `v0.6.0-alpha` (which is published from [tup-ui:/libs/core-ui](https://github.com/TACC/tup-ui/tree/tup-starter-nx--core-styles-integration)).